plugins/pastetools/filter/common.js

```diff
@@ -765,6 +765,191 @@
 
 	plug.parseShorthandMargins = parseShorthandMargins;
 
+	/**
+	 * Namespace containing all the helper functions to work with RTF.
+	 *
+	 * @private
+	 * @since 4.16.0
+	 * @member CKEDITOR.plugins.pastetools.filters.common
+	 */
+	plug.rtf = {
+		/**
+		 * Get all groups from the RTF content with the given name.
+		 *
```

Comment:
Can you add some super short sample here (with sample RTF) showing how it works? This RTF stuff could be hard to understand at first so some descriptive / easy to understand sample could be really helpful here.

Could you add similar samples to all 4 methods here?